js/menu.js

```diff
@@ -0,0 +1,76 @@
+// It uses `window.matchMedia` instead of `window.addEventListener('resize')` because `matchMedia` performs better by not changing its value with every screen resize.
+const mobileScreen = window.matchMedia("(max-width: 1110px)");
```

Comment:
What about moving these into css variables and grabbing from there? Same with `ismobile.js` so you have it all in one place? 
Just a suggestion but not a requirment. Having css is a file that is not being used there can get messy, but might be a way to have a single spot for the media settings.

Reply:
After looking again I'm seeing this is the only time you set something like this. I guess I was just seeing multiples due to the changes.

Reply:
Can CSS variables be brought into JavaScript?

Reply:
Yeah. With `getComputedStyle`. It'd be something like this, which could be made into a single helper function. All constants stored in the CSS.
  
```
# in CSS :root  { --max-width: 1110px} 

var style = getComputedStyle(document.body)
var width = style.getPropertyValue('--max-width') 
const mobileScreen = window.matchMedia($`(max-width:${max-width)`);
```

I just thought having constants in one place would be nice. This not a best practice I've seen used before though. Just an idea.

On another note, how about a new PR where we put the style.css re-used vals into variables? Since this file does not use them. I could work on this after my current issue is open?

Reply:
Thank you, I'm going to try it.

> how about a new PR where we put the style.css re-used vals into variables?

Are you referring to the reused values like the colors? If that's the case, i think it's a great idea, i recently opened an issue to define this, see #1628.

Reply:
Can CSS variables be used for the condition of a media query? I tried to add them, but I think it's not possible, I have to do things that I don't think are a good practice for it to work.
```
@media all and (max-width: var(--nav-mobile)) {
```

Maybe you can do it in another pr when we launch this?

Reply:
I think it’s good like u have it too.  So we can leave this idea for now.